domains/transfer-agreements/TransferAgreements.API/API.IntegrationTests/Testcontainers/WalletContainer.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Threading.Tasks;
+using DotNet.Testcontainers.Builders;
+using DotNet.Testcontainers.Containers;
+using Testcontainers.PostgreSql;
+using Xunit;
+
+namespace API.IntegrationTests.Testcontainers;
+
+public class WalletContainer : IAsyncLifetime
+{
+    private readonly Lazy<IContainer> walletContainer;
+    private readonly PostgreSqlContainer postgresContainer;
+
+    public WalletContainer()
+    {
+        postgresContainer = new PostgreSqlBuilder()
+            .WithDatabase("postgres")
```

Comment:
Isnt it set by default these values?

Reply:
You are right. I have removed the specification of the default values.